test/external/external_test_onnx_backend.py

```diff
@@ -164,6 +164,12 @@ def supports_device(cls, device: str) -> bool:
 backend_test.exclude('test_scatter_elements_with_reduction_max_cpu') # max not yet supported
 backend_test.exclude('test_scatternd_max_cpu') # max not yet supported
 
+# TODO: unsure why this is failing
+# 0: op "AveragePool" input shapes [(1, 1, 32, 32, 32)] opt {'ceil_mode': 1, 'count_include_pad': 1, 'dilations': (2, 2, 2),
+# 'kernel_shape': (5, 5, 5), 'strides': (3, 3, 3)}
+# torch doesn't support dilation but test vs torch passes when artificially expanding the kernel to (9, 9, 9)
```

Comment:
for the modified case that can be ran from torch 
```
x_shape = (32, 32, 32)
dilations = (1, 1, 1)
kernel_shape = (9, 9, 9)
strides = (3, 3, 3)
count_include_pad = 1
ceil_mode = True
```

using onnx's ground truth runner from https://github.com/onnx/onnx/blob/main/onnx/backend/test/case/node/averagepool.py#L560

it appears the onnx result differs from torch

what.